CHANGELOG.rst

```diff
@@ -0,0 +1,3 @@
+**v1.0.0 - mm/dd/yy**
```

Comment:
I had that originally since it matched our framework versioning but I noticed the default package version when using cookiecutter was 1.0.0. I'll change it to 0.0.0 for now and merge and we can change again if needed.